util/stmtsummary/statement_summary.go

```diff
@@ -55,6 +53,16 @@ type stmtSummaryByDigestMap struct {
 	// It's rare to read concurrently, so RWMutex is not needed.
 	sync.Mutex
 	summaryMap *kvcache.SimpleLRUCache
+
+	// enabledWrapper encapsulates variables needed to judge whether statement summary is enabled.
+	enabledWrapper struct {
+		sync.RWMutex
+		// enabled indicates whether statement summary is enabled in current server.
+		sessionEnabled string
+		// setInSession indicates whether statement summary has been set in any session.
+		globalEnabled  string
+		defaultEnabled bool
```

Comment:
I think we can remove `defaultEnabled`.
If the value is not `1`, then enabled is false.

Reply:
Default value should be defined once in the whole project so that it's convenient to modify.
Since `DefTiDBEnableStmtSummary` is already defined and is used in `variable.defaultSysVars`, we should reuse `DefTiDBEnableStmtSummary` as the default value in `stmtsummary`.

Reply:
The global default is used once it is loaded, the `DefTiDBEnableStmtSummary` is not needed.

Reply:
Addressed.